modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -182,18 +192,32 @@ protected void doServeResource(
 		}
 	}
 
-	private JSONArray _getAssetCategoriesJSONArray(
-		ContentDashboardItem contentDashboardItem, Locale locale) {
+	private HashMap<String, Object> _buildVocabularyDetails(
```

Comment:
We should never return the implementation of a Map, better if we use the interface. (You can check the differences betwen interfacex Map and the implementation HasMap in https://www.javatpoint.com/map-and-hashmap-in-java#:~:text=The%20Map%20is%20an%20interface%2C%20and%20HashMap%20is%20a%20class,by%20using%20its%20implementing%20classes.&text=Whereas%20HashMap%20implements%20Map%20interface,the%20Map%20and%20HashMap%20objects.)